src/main/groovy/com/inxmail/clicktracker/entity/Url.groovy

```diff
@@ -0,0 +1,19 @@
+package com.inxmail.clicktracker.entity
+
+import com.fasterxml.jackson.annotation.JsonFormat
```

Comment:
Interesting that a company named FasterXML provides *the* Java JSON library. :sunglasses: